src/index.js

```diff
@@ -49,6 +53,92 @@ const showPostsByTimeline = (timeline) => {
     })
 }
 
+/**
+ * This is a function to display name and description of author.
+ * @param {Object} profile - profile of the author
+ * @returns {null} null
+ */
+
+const showNameAndDesc = (profile) => {
+    log(title('name'))
+    log(body(profile.name));
+    log(title('\ndescription'))
+    log(body(`${profile.desc}\n`));
+}
+
+/**
+ * This is a function to display author profile details such as email,work,location,etc.
+ * @param {Object} profile - profile of the author
+ * @returns {null} null
+ */
+
+const showUserDetails = (profile) => {
+    profile.field.forEach((field, index) => {
+        log(title(field));
+        log(body(`${profile.value[index]}\n`));
+    });
+}
+
+/**
+ * This is a function to display the links of the author mentioned in profile.
+ * @param {Object} profile - profile of the author
+ * @returns {null} null
+ */
+
+const showLinks = (profile) => {
+    log(title('links'));
+    profile.links.forEach(link => {
+        log(body(link));
+    });
+}
+
+/**
+ * This is a function to display the details in sidebar such as skills/languages, projects and hacks, etc.
+ * @param {Object} profile - profile of the author
+ * @returns {null} null
+ */
+
+const showSidebarDetails = (profile) => {
+    log('');
+    profile.sidebarHeader.forEach((header, index) => {
+        log(title(header));
+        log(body(`${profile.sidebarBody[index]}\n`));
+    });
+}
+
+/**
+ * This is a function to display stats such as posts published, comments written, tags followed.
+ * @param {Object} profile - profile of the author
+ * @returns {null} null
+ */
+
+const showStats = (profile) => {
+    log(title('stats'));
+    profile.stats.forEach(stat => {
+        log(body(stat));
+    });
+}
+
+/**
+ * This is a driver function to call all the above functions that display profile details on terminal.
+ * @param {Object} profile - profile of the author
+ * @returns {null} null
+ */
+
+const showUserProfile = (username) => {
+    crawler.fetchAuthorProfile(username).then((profile) => {
+        if (!profile.name) {
+            console.error("😱 User not found. Please try again.");
+            process.exit(1);
+        }
+        showNameAndDesc(profile);
+        showUserDetails(profile);
+        showLinks(profile);
+        showSidebarDetails(profile);
+        showStats(profile);
+    });
+}
+
```

Comment:
Well done 👍🏻 !! but It's too much code to put in the main file. Please make a module in the util folder and then include that module here. See how I have done in the case of Prompts and Crawler.

Reply:
Also, Try to put a loader as well.

Reply:
Thanks for the review. Have made the changes as asked. I'm still learning to write quality code, so if there are any more changes required, kindly let me know.

Reply:
You did a great job dude. 😊
Will let you know if I found anything else.